src/components/HorizontalList/HorizontalListHome/HorizontalListHome.tsx

```diff
@@ -0,0 +1,71 @@
+import { HorizontalList } from "../HorizontalList";
+export function HorizontalListHome() {
```

Comment:
Let's make this component more generic by not calling it `HorizontalListHome`, but rather something like plain `HorizontalList`.